agenthub/coact_agent/meta/executor_prompt.py

```diff
@@ -0,0 +1,37 @@
+EXECUTOR_PASS_CHECK = """Now, your role is to ensure the successful execution of actions in the global plan. Verify the results of these actions and compare them to the global plan. If they align, proceed to the next phase and output the action decision as ```move```. If discrepancies arise, you have two options:
+1) If you suspect issues with your local plan, output the action ```revise```, or
+2) If you suspect problems with the global planner's plan, trigger a request for replanning by outputting the action ```request```.
+If the actions align with the global plan, explain the reasons for this alignment. If discrepancies arise, provide detailed reasons for your action decision:
```

Comment:
Nit: maybe we can say action or decision, instead of both? ("your action decision")

Reply:
I still haven't looked into it 😅 But at fist glance I think the original impl has quite many actions and other things like "local plan", which may make things over-complicated. Perhaps we may want to try with a subset of them first: only global plan, no local plan (hence no "revise" for executor, only "request")? Wdyt?

Reply:
That makes sense to me... Feels like a relevant change though. Did the tests so far use 'revise' in practice?

Reply:
No, only "request for a new plan"

Reply:
Then we could simplify it, if you think it's best. I would still suggest to remove the "Page Operations Actions" too if we're at it, since it refers to... nothing here?

Reply:
Sure, removed the whole meta prompts folder and will add back later.